uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -73,28 +73,101 @@ public function testCanTransitionToForLeaveApproverReturnsFalseForAllNonPossible
   }
 
   public function testCanTransitionToForAdminForReturnsTrueAllPossibleTransitionStatuses() {
+    $adminID = 5;
```

Comment:
I can see how it might run faster but I think that tests should be responsible for creating and then destroying all data they use. If tests rely on data created by other tests it could lead to one change breaking lots of tests in other extensions.